setup/install/providers/kubernetes-v2/aws-eks.md

```diff
@@ -49,35 +39,33 @@ EKS_CLUSTER_CA_DATA=$(aws cloudformation describe-stacks --stack-name spinnaker-
 SPINNAKER_INSTANCE_PROFILE_ARN=$(aws cloudformation describe-stacks --stack-name spinnaker-managing-infrastructure-setup --query 'Stacks[0].Outputs[?OutputKey==`SpinnakerInstanceProfileArn`].OutputValue' --output text)
 ```
 
+## Set up the managed account
 
-## Managed Account - Base setup
-
-In each of managed account, create role that can be assumed by Spinnaker
+In each of managed accounts, create a IAM role that can be assumed by Spinnaker:
 
 > This needs to be executed in managing account as well.
 
 ```bash
-curl -O https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/managed.yaml  
+curl -O https://www.spinnaker.io/setup/install/providers/aws/managed.yaml  
 
 aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --template-file managed.yaml \
 --parameter-overrides AuthArn=$AUTH_ARN ManagingAccountId=$MANAGING_ACCOUNT_ID --capabilities CAPABILITY_NAMED_IAM
 ```
 
-## kubectl configurations
+## `kubectl` configurations
 
-Before you proceed next, ensure to configure [kubectl and heptio authenticator for aws is installed and configured](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html)
+Before you proceed,configure [kubectl and heptio authenticator for aws is installed and configured](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html)
```

Comment:
I think this should be step 1. 
Also, "...configure...is installed and configured." (I think it's still broken.)

If you make this step one, you can remove "Before you proceed"
How about this:
"1. Configure `kubectl` and Heptio Authenticator for AWS."